desc/objectives/_generic.py

```diff
@@ -351,17 +351,17 @@ def myfun(grid, data):
             # the unique values:
             return grid.compress(f_fsa)
 
-        myobj = ObjectiveFromUser(myfun)
+        myobj = ObjectiveFromUser(fun=myfun, thing=eq)
 
     """
 
     _units = "(Unknown)"
-    _print_value_fmt = "Custom Objective value: {:10.3e}"
+    _print_value_fmt = "Custom objective value: {:10.3e}"
 
     def __init__(
         self,
         fun,
-        eq,
+        thing,
```

Comment:
im not crazy about changing the name of arguments without a deprecation warning etc, so that we don't suddenly break user code.

I'd recommend changing the name, but add `**kwargs` and check if `eq` is in kwargs and throw a warning in that case. See https://github.com/PlasmaControl/DESC/blob/f955d3de2be694618593057ea9c0585595255ef2/desc/utils.py#L492-L493